plugins/chat/spec/system/drawer_spec.rb

```diff
@@ -12,6 +12,20 @@
     chat_page.prefers_drawer
   end
 
+  it "handles transitions between drawer and full page and applies appropriate classes" do
+    visit("/")
+    chat_page.open_from_header
+    expect(page).to have_css(
+      "body.has-drawer-chat.has-chat.chat-drawer-active.chat-drawer-expanded",
+    )
+    expect(page).to have_css("html.has-drawer-chat.has-chat")
+    expect(page).to have_no_css("body.has-full-page-chat")
+    drawer_page.maximize
+    expect(page).to have_css("body.has-chat.has-full-page-chat")
+    expect(page).to have_css("body.has-chat.has-full-page-chat")
```

Comment:
```suggestion
    expect(page).to have_css("body.has-chat.has-full-page-chat")
```

Removing duplicated line.